mintapi/api.py

```diff
@@ -1130,8 +1133,12 @@ def main():
     cmdline.add_argument(
         '--mfa-method',
         default='sms',
-        choices=['sms', 'email'],
+        choices=['sms', 'email', 'soft-token'],
         help='The MFA method to automate.')
+    cmdline.add_argument(
+        '--mfa-token',
+        default=None,
+        help='The MFA soft-token to pass to oathtool.')
```

Comment:
If mfa-token logically implies the method of soft token, I'm fine with making that implicit. Your call!

Reply:
Right now you would have to pass:
```
--mfa-method soft-token --mfa-token $token
```

We could change it to just:
```
--mfa-token $token
```

Perhaps another idea is something like this if it's possible:
```
--mfa-method soft-token=$token
```

I'm not sure what is the most idiomatic style for python or this library.

Reply:
I think that's fine if that seems simplest! If the mfa-method was implicit if you passed a token that would also be fine.